src/features/common/FirePopup/index.tsx

```diff
@@ -84,7 +128,7 @@ export default function FirePopup({ isOpen }: Props) {
         onMouseLeave={() => setShowPopup(false)}
         className={styles.fireIcon}
       >
-        <FireIcon width={36} />
+        <FireIcon width={24} height={28} />
```

Comment:
You can remove the height prop from the FireIcon component , it won't have any effect in the UI